src/Controllers/ShareDraftController.php

```diff
@@ -157,7 +157,11 @@ private function getRenderedPageByURL(string $url): HTTPResponse
         $variables = HTTPRequestBuilder::cleanEnvironment(Environment::getVariables());
         $variables['_SERVER']['REQUEST_URI'] = $url;
         $variables['_SERVER']['REQUEST_METHOD'] = 'GET';
-        $variables['_SERVER']['HTTP_USER_AGENT'] = 'CLI';
+        $variables['_SERVER']['HTTP_USER_AGENT'] = 
```

Comment:
```suggestion
        $variables['_SERVER']['HTTP_USER_AGENT'] =
```
Will fix the PHPCS issue in travis

Reply:
Have tested locally, works and also tested separately successfully with `axllent/silverstripe-trailing-slash:^2` installed

Happy to merge once the PHPCS issue in travis is resolved